src/main/java/com/amazon/ion/impl/IonRawWriter_1_1.kt

```diff
@@ -65,6 +65,12 @@ interface IonRawWriter_1_1 {
      */
     fun _private_hasFirstAnnotation(sid: Int, text: String?): Boolean
 
+    /**
+     * Writes one annotation for the next value.
+     * [writeAnnotations] may be called more than once to build up a list of annotations.
+     */
+    fun writeAnnotations(annotation0: SystemSymbols_1_1)
```

Comment:
Nit: reading this I was a bit thrown off by `writeAnnotations` being plural, `annotation0` being singular, and `SystemSymbols_1_1` being plural. Could this flavor of the method have a singular name?
```kt
writer.writeAnnotation(SystemSymbols_1_1.ION_ENCODING)
```
I see it's part of an interface--if changing it comes with a lot of baggage then just leave it.

Reply:
I intentionally made it plural because it overloads the existing `writeAnnotations` methods which are intended to provide a pseudo-varags API without the overhead of varargs.
```kotlin
    fun writeAnnotations(annotation0: Int)
    fun writeAnnotations(annotation0: Int, annotation1: Int)
    fun writeAnnotations(annotations: IntArray)
    fun writeAnnotations(annotation0: CharSequence)
    fun writeAnnotations(annotation0: CharSequence, annotation1: CharSequence)
    fun writeAnnotations(annotations: Array<CharSequence>)
```

I'm fine with changing it, but I'd prefer deal with the naming later—perhaps as part of a larger-scale review/revisiting of new API names/signatures before merging the Ion 1.1 branch into the main branch.